### ISSUE TITLE
GitHub pages not working

### ISSUE THREAD

**Issue by Siu771** [Tacogamerman/Calculator#14]
GitHub pages is not working, it is showing as 404 file not found.

**Comment by mileswolfallen3** [comment]
ok i will chek this out

**Comment by mileswolfallen3** [comment]
ps this is mileswa1q2 alt

**Comment by Tacogamerman3** [comment]
> ok i will chek this out

Blud you are NOT me

**Comment by PS50YT** [comment]
**Solution** 
Rename home page as "index.html"

**Comment by luminosity-d3v** [comment]
@PS50YT You're not wrong. Ideally, making the "home page" branch or whatever it is called would theoretically make it usable for Github Pages, making it an infinitely duplicateable link. 
@mileswolfallen3 Don't bother faking it man; you failed to even put the real name in 💀
